src/test/java/uk/gov/hmcts/auth/provider/service/api/microservice/MicroserviceRepositoryTest.java

```diff
@@ -27,19 +27,20 @@ public void returnsMicroserviceCaseInsensitive() {
     }
 
     @Test
-    public void getNames_should_return_names_of_configured_services() {
+    public void getNames_should_return_names_of_configured_services_in_alphabetical_order() {
```

Comment:
While you at this line, do we need to remove `getNames_` prefix from our _usual_ scenario naming?

Reply:
`MicroserviceRepository` contains more than repo, this clarifies which method we're testing.